src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx

```diff
@@ -1,83 +1,62 @@
 import ButtonLink from '@admin/components/ButtonLink';
-import Page from '@admin/components/Page';
-import usePublicationContext from '@admin/contexts/PublicationContext';
 import {
   legacyReleaseCreateRoute,
   legacyReleaseEditRoute,
 } from '@admin/routes/legacyReleaseRoutes';
-import { dashboardRoute, PublicationRouteParams } from '@admin/routes/routes';
+import { PublicationRouteParams } from '@admin/routes/routes';
 import legacyReleaseService, {
   LegacyRelease,
 } from '@admin/services/legacyReleaseService';
-import publicationService from '@admin/services/publicationService';
-import appendQuery from '@admin/utils/url/appendQuery';
+import publicationService, {
+  BasicPublicationDetails,
+} from '@admin/services/publicationService';
 import Button from '@common/components/Button';
 import ButtonGroup from '@common/components/ButtonGroup';
-import LoadingSpinner from '@common/components/LoadingSpinner';
 import ModalConfirm from '@common/components/ModalConfirm';
 import useToggle from '@common/hooks/useToggle';
 import reorder from '@common/utils/reorder';
+import styles from '@admin/pages/legacy-releases/components/LegacyReleasesTable.module.scss';
```

Comment:
Does it make sense to rehome this css module under a different folder now?

`pages/legacy-releases` suggests that it still belongs to a "Legacy Releases" page.

Reply:
It should be in the same folder as the Table component. Though thinking about it, maybe that should be under publication instead as it's used on the edit publication page.